addOns/graphql/src/main/javahelp/org/zaproxy/addon/graphql/resources/help/contents/automation.html

```diff
@@ -21,6 +21,7 @@ <H2>Job: graphql</H2>
       endpoint:                        # String: the endpoint URL, default: null, no schema is imported
       schemaUrl:                       # String: URL pointing to a GraphQL Schema, default: null, import using introspection on endpoint
       schemaFile:                      # String: Local file path of a GraphQL Schema, default: null, import using schemaUrl
+      queryGenEnabled:                 # Bool: Whether the Query Generator is enabled, default: true
```

Comment:
I don't think it's true by default, if you disable it in the options and don't provide the field the queries are not generated. In the job dialogue the field is also shown as deselected.